ui/pages/home/home.component.js

```diff
@@ -188,6 +214,55 @@ export default class Home extends PureComponent {
     }
   }
 
+  ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
+  checkInstitutionalConnectRequest() {
```

Comment:
Can we have some tests for this function?

Reply:
Not much time atm, but we will come back to this afterwards for sure, as this will still be subject to change by us.
Will create a ticket for it 👍🏼